lang/node-i2c-bus/Makefile

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (C) 2016 ?
```

Comment:
@openwrt/package-maintainers 

Well, I am also not sure what to put in the copyright header now.  Is it fine that authors and contributors of the file hold the copyright as long as it is declared as licensed under GPLv2 and has a signed-off-by line?